qa/common/util.sh

```diff
@@ -234,6 +234,46 @@ function run_server_nowait () {
     SERVER_PID=$!
 }
 
+# Run inference server inside a memory management tool like Valgrind/ASAN. 
+# Return once server's health endpoint shows ready or timeout expires. Sets 
+# SERVER_PID to pid of SERVER, or 0 if error (including expired timeout)
+function run_server_leakcheck () {
```

Comment:
You may want to make this utility for only running the server, like `run_server`, the caller should kill the server explicitly. Because we may want to collect valgrind result of different server states (i.e. running n inferences, or invoke all endpoints...)